source/backend/api/Mapping/RegionMap.cs

```diff
@@ -0,0 +1,16 @@
+using Mapster;
+using Pims.Api.Models;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Mapping
+{
+    public class RegionMap : IRegister
```

Comment:
Region inherits from ICodeEntity so you may not need this. See AddressMap or  LookupMap

Reply:
Updated